gender-yearsExperience.sh

```diff
@@ -0,0 +1,5 @@
+#sort wages.csv by gender then years experience
```

Comment:
When provided, comments in code are descriptive. Please comment code consistently across script. 
Work on providing more descriptive commit messages for collaborators and future you :)

Reply:
2.5/3